LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-duplicate-names-without-enforcement.html

```diff
@@ -6,6 +6,10 @@
 <body>
 <script>
 const policy = { createHTML: a => a };
+const policy_default = { createHTML: (a, _, sink) => {
+  assert_equals(sink, 'Element innerHTML');
+  return a;
+}};
```

Comment:
Similarly to what @mbrodesser-Igalia  commented above, this one will never be executed in that test. Fortunately, this sink is already covered by your change from trusted-types-createHTMLDocument.html above.

Anyway, I think we should revert the change to trusted-types/trusted-types-duplicate-names-without-enforcement.html here.

cc @lukewarlow